generators/chipyard/src/main/scala/upf/UPFInputs.scala

```diff
@@ -0,0 +1,29 @@
+package chipyard.upf
+
+
+object UPFInputs {
```

Comment:
Need a comment probably in this file indicating that the output file(s) are dumped in the `generated-src/long-name` folder (also a sentence on the outputs in the documentation)